caas/kubernetes/provider/bootstrap.go

```diff
@@ -1361,6 +1409,25 @@ func (c *controllerStack) controllerContainers(jujudCmd, controllerImage string)
 	return containerSpec, nil
 }
 
+func jujudPebbleLayer(machineCmd string, env map[string]string) ([]byte, error) {
+	layer := plan.Layer{
```

Comment:
I think we should add in the jujud health checks here.

Let me know if you need a hand getting this setup.

Reply:
To be added in a future PR